docs/esps/index.rst

```diff
@@ -35,33 +36,33 @@ The table below summarizes the Anymail features supported for each ESP.
 
 .. rst-class:: sticky-left
 
-============================================  ============  =======  ============  ===========  ==========  ===========  ==========  ==========  ========  ==========  ===========
-Email Service Provider                        |Amazon SES|  |Brevo|  |MailerSend|  |Mailgun|    |Mailjet|   |Mandrill|   |Postal|    |Postmark|  |Resend|  |SendGrid|  |SparkPost|
-============================================  ============  =======  ============  ===========  ==========  ===========  ==========  ==========  ========  ==========  ===========
+============================================  ============  =======  ============  ===========  ==========  ===========  ==========  ==========  ========  ==========  ==============  ===========
+Email Service Provider                        |Amazon SES|  |Brevo|  |MailerSend|  |Mailgun|    |Mailjet|   |Mandrill|   |Postal|    |Postmark|  |Resend|  |SendGrid|  |Unisender Go|  |SparkPost|
```

Comment:
Please keep ESP names in alphabetic order (so "Unisender Go" would come last in the current list, after "SparkPost").

(Also, sorry this table is so hard to edit.)